File: algorithm/typescript/BOJ1152.ts
// problem : https://www.acmicpc.net/problem/1152

import fs = require("fs");

const input: string[] = fs.readFileSync("/dev/stdin").toString().trim().split(" ");
let answer: number = 0;

console.log(input);
for (let i = 0, size = input.length; i < size; i++) {
  if (input[i] === "") {
    continue;
  }
  answer++;
}

console.log(answer);

File: algorithm/java/src/BOJ1152.java
// problem : https://www.acmicpc.net/problem/1152

import java.io.BufferedReader;
import java.io.InputStreamReader;

public class BOJ1152 {
    static String[] arr;

    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        arr = br.readLine().split("");

        int answer = 0;
        for (int i = 0; i < arr.length; i++) {
            if (arr[i].equals("")) {
                continue;
            }

            answer++;
        }

        System.out.println(answer);
    }
}
